lbry/tests/integration/test_claim_commands.py

```diff
@@ -666,6 +666,49 @@ class StreamCommands(ClaimTestCase):
         signed = await self.out(self.stream_create('bar', '1.0', channel_name='@spam', preview=True, confirm=False))
         self.assertTrue(signed['outputs'][0]['is_channel_signature_valid'])
 
+    async def test_repost(self):
+        await self.out(self.channel_create('@goodies', '1.0'))
+        tx = await self.out(self.stream_create('newstuff', '1.1', channel_name='@goodies'))
+        claim_id = tx['outputs'][0]['claim_id']
+        await self.stream_repost(claim_id, 'newstuff-again', '1.1')
+        claim_list = await self.out(self.daemon.jsonrpc_claim_list())
+        reposts_on_claim_list = [claim for claim in claim_list if claim['value_type'] == 'repost']
+        self.assertEqual(len(reposts_on_claim_list), 1)
+        await self.out(self.channel_create('@reposting-goodies', '1.0'))
```

Comment:
`channel_create` already runs the result through `self.out` I believe, no need to do it again.